scripts/create-package/README.md

```diff
@@ -0,0 +1,4 @@
+# `create-package`
```

Comment:
Aligns with core to enable a CLI to create another package we will likely want to appropriate this core `create-component`